django_tables2/columns/booleancolumn.py

```diff
@@ -58,6 +59,6 @@ def render(self, value, record, bound_column):
     @classmethod
     def from_field(cls, field):
         if isinstance(field, models.BooleanField):
-            return cls(verbose_name=field.verbose_name, null=False)
+            return cls(verbose_name=title(field.verbose_name), null=False)
         if isinstance(field, models.NullBooleanField):
-            return cls(verbose_name=field.verbose_name, null=True)
+            return cls(verbose_name=title(field.verbose_name), null=True)
```

Comment:
I just tried this and it failed a test. Unfortunately I don't think we simplify by extracting the title call. Do you want the title call assigned to a local variable before constructing the column class in these or was this note just for the booleancolumn since it had two calls and could potentially be simplified?

```
    @classmethod
    def from_field(cls, field):
        verbose_name = title(field.verbose_name)
        if isinstance(field, models.BooleanField):
            return cls(verbose_name=verbose_name, null=False)
        if isinstance(field, models.NullBooleanField):
            return cls(verbose_name=verbose_name, null=True)
```

```
___________________________________________________________________________ test_foreign_key ___________________________________________________________________________

    def test_foreign_key():
>       class PersonTable(tables.Table):

tests/test_models.py:367: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
django_tables2/tables.py:197: in __new__
    extra[field_name] = columns.library.column_for_field(field)
django_tables2/columns/base.py:40: in column_for_field
    column = candidate.from_field(field)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

cls = <class 'django_tables2.columns.booleancolumn.BooleanColumn'>, field = <django.contrib.contenttypes.fields.GenericForeignKey object at 0x7f13a8b9aa90>

    @classmethod
    def from_field(cls, field):
>       verbose_name = title(field.verbose_name)
E       AttributeError: 'GenericForeignKey' object has no attribute 'verbose_name'
```